app/Integrations/Slack/SlackPlugin.php

```
@@ -0,0 +1,230 @@
+                'domain' => 'reaction',
+                'action' => 'added',
+                'value' => 1,
+                'value_unit' => 'reaction',
+                'event_metadata' => [
+                    'reaction' => $event['reaction'],
+                ],
+            ]],
+        ];
+    }
+    
+    protected function convertFileEvent(array $data, Integration $integration): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$integration'. (Unused Code Rules)

(UnusedFormalParameter)